docs/src/main/asciidoc/writing-extensions.adoc

```diff
@@ -538,6 +541,42 @@ So you may consider omitting explicit `name` in some cases.
 // The following link should point to the mojo page once https://github.com/quarkusio/quarkusio.github.io/issues/265 is fixed
 Please refer to https://github.com/quarkusio/quarkus/blob/{quarkus-version}/devtools/maven/src/main/java/io/quarkus/maven/CreateExtensionMojo.java[CreateExtensionMojo JavaDoc] for all the available options of the mojo.
 
+==== Using Gradle
+
+You will need to apply the `io.quarkus.extension` plugin in your `runtime` module and declare `quarkus-extension-processor` as annotation processor:
```

Comment:
```suggestion
You will need to apply the `io.quarkus.extension` plugin and enable annotation processor `io.quarkus:quarkus-extension-processor` in the `runtime` module of your extension project:
```

Reply:
BTW, the annotation processor could be added by the plugin itself, right?

Reply:
Are recommending to `enforcePlatform("io.quarkus:quarkus-bom:version")` somewhere?

Reply:
yes, for the runtime module I will push a commit to add it, however, for the deployment module, it could be tricky to find the module.

Reply:
I will add a `platform("io.quarkus:quarkus-bom:version")` reference in example. We can't use `enforcedPlatform` as it will block the publish.

Reply:
Aren't you generating the `META-INF/quarkus-extension.properties` with the deployment artifact coordinates? Couldn't we find the deployment subproject and enable the annotation processor in it?

Reply:
Yes, and this is a property of the first task, maybe I could move it to an extension to have global plugin configuration available a configuration time.